### PR TITLE
blacklists the blob event from several gamemodes

### PR DESCRIPTION
**Pull request by Blute1**
Blob is now blacklisted on: blob (obviously), nuke ops, clock cult, cult, and malfunction. 

:cl: Spacedong
tweak: The random blob event is now blacklisted from several gamemodes: Nuclear Emergency, Cult, Clockwork Cult, Malfunction, and Revolution.
/:cl:

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)

These gamemodes have specific, overarching antagonists that the crew must band together to defeat, and having a random giant blob bursting into the station halting the shuttle from escaping and consuming the station interrupts the flow of the gamemodes.

### CODE REVIEW

File: hippiestation/code/modules/events/blob.dm [hunk 1]
```diff
@@ -0,0 +1,2 @@
+/datum/round_event_control/blob
```
Comment (ghost) [inline hippiestation/code/modules/events/blob.dm:1]: You need to add this file to the DME

**Comment by Dudeinyourbackyard** [comment]
ye its weird just having it say hello in random rounds
+1

**Comment by dreamcrab** [comment]
fuck it why not +d0ng2456

**Comment by bardofrage** [comment]
+1

**Comment by Blute1** [comment]
3 headmin +1's

**Comment by PopNotes** [comment]
reviewers aren't supposed to merge but it's literally three line changes who gives a fuck

**Pull request merged**